### DISCUSSION TITLE
Change the name in my github user profile

### DISCUSSION THREAD

**Discussion by kinnrai** [community/community#52888] [GitHub Education] [answered]
I tried many times, but I was unable to pass the verification until I changed my Github username to my real name and added a bio. 

Luckily, I finally succeeded in passing!

However, now I'm curious if I'm able to change my username back to my nickname because I don't want to reveal my real name. Will changing my username cause me to lose Github Student Verification status?

**Comment by pnpn111** [comment, marked as answer]
Yes, changing your name back to your previous name after the application is successful will not affect your application status. I've already done that.

**Reply by samus-aran** [reply]
Thanks @pnpn111 for sharing that confirmation 👏 👏 🎉

**Reply by satyamadugulla** [reply]
I don't reveal name M sacred hacker after application status series thanks for teaching
